zerver/webhooks/travis/doc.md

```diff
@@ -0,0 +1,22 @@
+See your Travis CI build notifications in Zulip!
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+Finally, set up a webhook in your `.travis.yml` file:
+
+```
+notifications:
+webhooks:
+- {{ external_api_uri_subdomain }}/v1/external/travis?stream=travis&amp;topic=build-status&amp;api_key=abcdefgh
+```
+
+By default, pull request events are ignored since most people
+don't want notifications for new pushes to pull requests.  To
+enable notifications for pull request builds, just
+add `&ignore_pull_requests=false` at the end of the url.
```

Comment:
Too late. I'll create a commit to fix this as @timabbott already merged the Travis one.